lib/tests/unit/modules/command_test.cc

```diff
@@ -209,45 +218,3 @@ TEST_CASE("Modules::Command::callAction where the command fails", "[modules]") {
         }
     }
 }
-
-TEST_CASE("Modules::Command::buildCommandObject where the executable contains spaces", "[modules]") {
```

Comment:
Is it not worth testing commands with spaces in them anymore?

Reply:
Not anymore no. The command no longer needs to be split based on which string should go in the "Executable" part since the actual executable is the shell interpreter, and everything else including the command to run is an argument.